server/monitor.go

```diff
@@ -752,6 +752,8 @@ func (s *Server) HandleRoutez(w http.ResponseWriter, r *http.Request) {
 
 // Subsz represents detail information on current connections.
 type Subsz struct {
+	ID  string    `json:"server_id"`
```

Comment:
Just FYI we need to add in server_name as well here for all of these. server_name survives if set, id is ephemeral.

Reply:
Will add that when adding accounts options. Next on my list

Reply:
Will add that in concert with another change